astropy/units/astrophys.py

```diff
@@ -26,7 +26,7 @@
 ###########################################################################
 # LENGTH
 
-def_unit((['AU', 'au'], []), _si.au.value * si.m, namespace=_ns, prefixes=True,
+def_unit((['AU', 'au'], ['AstronomicalUnit']), _si.au.value * si.m, namespace=_ns, prefixes=True,
```

Comment:
I'll throw in my vote for `astronomical_unit` as well, for the reason that most other units are lower case unless they are named after a person.